CODE_OF_CONDUCT.md

```diff
@@ -18,7 +18,8 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
 community include:
 
-* Demonstrating empathy and kindness toward other people
```

Comment:
CODE_OF_CONDUCT.md is taken from Contributor Covenant, version 2.1. It is not good for maintenance when upgrading Contributor Covenant to change the wording of the quotation, so it is preferable to leave it as is.

Reply:
@tkoyama010, do you have a suggestion on where we could put this? Perhaps in the CONTRIBUTING.md guide?

Reply:
@banesullivan Agree. CONTRIBUTING.md is a good place to add this.